vnext/Microsoft.ReactNative/Modules/PaperUIManagerModule.cpp

```diff
@@ -323,8 +329,9 @@ class UIManagerModule : public std::enable_shared_from_this<UIManagerModule>, pu
   }
 
   void sendAccessibilityEvent(int64_t reactTag, double eventType) noexcept {
-    assert(false);
-    // TODO
+    if (eventType == 0) { // FocusAccessibilityEvent Type
```

Comment:
I see that in RN this value is coming from
https://github.com/facebook/react-native/blob/6e6443afd04a847ef23fb6254a84e48c70b45896/ReactAndroid/src/main/java/com/facebook/react/uimanager/UIManagerModuleConstants.java#L116-L117

Do we expect apps to ever use the int value instead of the string? if so, the value should be 8:

https://www.android-doc.com/reference/android/view/accessibility/AccessibilityEvent.html#TYPE_VIEW_FOCUSED